lib/authz/permissions.go

```diff
@@ -70,7 +81,14 @@ type AuthorizerOpts struct {
 	// DisableDeviceAuthorization disables device authorization via [Authorizer].
 	// It is meant for services that do explicit device authorization, like the
 	// Auth Server APIs. Most services should not set this field.
+	//
+	// DisableDeviceAuthorization is equivalent to both
+	// DeviceAuthorization.DisableGlobalAuthz=true and
+	// DeviceAuthorization.DisableRoleAuthz=true.
 	DisableDeviceAuthorization bool
```

Comment:
Any reason not to delete this bool and just use the DeviceAuthorizationOpts? I'd think it would be better to have one way of doing things

Reply:
Done.